universal-application-tool-0.0.1/app/services/question/QuestionDefinition.java

```diff
@@ -16,14 +19,15 @@
   private final ImmutableMap<Locale, String> questionText;
   private final Optional<ImmutableMap<Locale, String>> questionHelpText;
 
+  @JsonCreator
   public QuestionDefinition(
-      String id,
-      String version,
-      String name,
-      String path,
-      String description,
-      ImmutableMap<Locale, String> questionText,
-      Optional<ImmutableMap<Locale, String>> questionHelpText) {
+      @JsonProperty("id") String id,
```

Comment:
suggestion (non-blocking): may as well change ID to a `long` now since it'll soon be coming from the DB as an auto incrementing long.

Reply:
Done